tests/test_json_writer.py

```diff
@@ -246,6 +246,7 @@ def test_GIVEN_stream_in_group_children_WHEN_handling_group_THEN_stream_is_appen
     converter = NexusToDictConverter()
     root_dict = converter.convert(file)
 
+    assert len(root_dict["children"]) == 1
```

Comment:
It would be good to explain why you are testing this. You can use `assert <condition>, "some string"`